environment.yml

```diff
@@ -62,11 +62,13 @@ dependencies:
   - django-model-utils
   - django-guardian
   - django-gravatar2
+  - django-json-widget
   - django-mfa2
   - django-recaptcha2
   - django-simple-captcha
   - django-session-security
   - django-termsandconditions
+  - django-json-widget
```

Comment:
django-json-widget is in this file twice. Remove the second one.

Reply:
Done